### PR TITLE
[IMP] l10n_in_edi: error for partner with it's name

### PR DESCRIPTION
**Pull request by suva-odoo**
Before this commit
================
Before only error message was displayed, but not the partner's name for which it was causing error

After this commit
===============
After changes the partner's name is displayed along with the error message

task-3293271

### CODE REVIEW

File: addons/l10n_in_edi/models/account_edi_format.py [hunk 1]
```diff
@@ -239,6 +239,8 @@ def _l10n_in_validate_partner(self, partner, is_company=False):
             or not re.match("^.{6,100}$", partner.email)
         ):
             message.append(_("\n- Email address should be valid and not more then 100 characters"))
+        if message:
+            message.insert(0, "\n%s" %(partner.name))
```
Comment (jva-odoo) [inline addons/l10n_in_edi/models/account_edi_format.py:243]:
```suggestion
            message.insert(0, "\n%s" %(partner.display_name))
```

**Review by jva-odoo — APPROVED** [review]
@jco-odoo LGTM

**Comment by suva-odoo** [comment]
error_msg

File: addons/l10n_in_edi/models/account_edi_format.py [hunk 2]
```diff
@@ -239,6 +239,8 @@ def _l10n_in_validate_partner(self, partner, is_company=False):
             or not re.match("^.{6,100}$", partner.email)
         ):
             message.append(_("\n- Email address should be valid and not more then 100 characters"))
+        if message:
+            message.insert(0, "\n%s" %(partner.display_name))
```
[the remark this answers is not in the document]
Comment (jco-odoo) [reply addons/l10n_in_edi/models/account_edi_format.py:243]: Could we just remove all the \n int he strings here?

**Comment by jco-odoo** [comment]
@suva-odoo @jva-odoo I suppose we can merge if you squash the 2 commits into 1?

**Pull request closed without merging**